.gitignore

```diff
@@ -1 +1,2 @@
-venv/
\ No newline at end of file
+venv/
+**__pycache__/
```

Comment:
How is this related to setting up code quality monitoring ?

Reply:
> How is this related to setting up code quality monitoring ?

I take it this part should go in a different pull request? I'm gonna have to fix that on some other PRs too then.